list_filter.py

```diff
@@ -0,0 +1,76 @@
+# Python built-in imports.
+import os
+import urllib
+import logging
+import webapp2
+import jinja2
+
+# Third party imports.
+from google.appengine.ext import ndb
+
+# First party imports
+from warehouse_models import Item
+import auth
+import list_filter
+
+## Global Variables
+name_filter = ""
```

Comment:
I am worried about using globals. While it may work locally it will not work when multiple users are using the site or when there are multiple copies of the server. I would suggest that you don't worry about persisting the user's selections and we can worry about that in the future. I think a good way to do it could be through using cookies but we can talk about it more in future meetings.

Reply:
Cool. I'll wait until Friday to learn about cookies from you and will fix this as soon as possible.